amundsen_application/api/metadata/v0.py

```diff
@@ -388,26 +457,26 @@ def _log_update_table_tags(*, table_key: str, method: str, tag: str) -> None:
         args = request.get_json()
         method = request.method
 
-        table_endpoint = _get_table_endpoint()
         table_key = get_query_param(args, 'key')
 
         tag = get_query_param(args, 'tag')
 
-        url = f'{table_endpoint}/{table_key}/tag/{tag}'
-
         _log_update_table_tags(table_key=table_key, method=method, tag=tag)
 
-        response = request_metadata(url=url, method=method)
-        status_code = response.status_code
+        metadata_status_code = _update_metadata_service(table_key=table_key, method=method, tag=tag)
+        search_status_code = _update_search_service(table_key=table_key, method=method, tag=tag)
```

Comment:
what happens if the search doesn't have the endpoint?

Reply:
Could you clarify a little bit how could this happen?

Reply:
never mind, I think I misunderstood the part initially.